tools/clang/test/HLSLFileCheck/validation/NonVLAArrays.hlsl

```diff
@@ -0,0 +1,15 @@
+// RUN: %dxc -E PSMain -T ps_6_0 %s | FileCheck %s
+
+// CHECK: error: variable length arrays are not supported in HLSL
+// CHECK-NEXT: float4 BrokenTestArray0[int(.3)*12];
```

Comment:
If we are going to check for this failure, it would be better to check it independently of the checks for lack of failure. If the compiler failed due to this failure and also produced failures for the min and or the max, then this test would still pass. You could conditionally include the expected failure array or you could split this file into one that passes and one that fails. 

That said, I'm not sure we landed on wanting this to fail.